### PR TITLE
DoControl integration with DDOG (for integration extras tile)

### PR DESCRIPTION
**Pull request by grosenthal1**
### What does this PR do?

DoControl integration tile for DDOG integration extras

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:5]:
```suggestion
This integration allows [DoControl](https://www.docontrol.io/) customers to forward their DoControl related logs and events to Datadog through automated security workflows.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:9]:
```suggestion
To set up this integration, you must have an active [DoControl account](https://www.docontrol.io/demo). You must also have proper admin permissions in Datadog.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:15]:
```suggestion
### Use Datadog actions in DoControl's workflows
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:17]:
```suggestion
You must create a Datadog API key and an application key to use as input parameters for Datadog actions in DoControl.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:21]:
```suggestion
Be sure to copy and save your API Key as you will not be able to access it after exiting the page. For more information, see [API and Application Keys](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token).
```
I would also move this section to underneath step 5.

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:23]:
```suggestion
1. In Datadog, hover over your username in the bottom left corner and select **Organization Settings**.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:37]:
```suggestion
Be sure to copy and save your Application Key as you will not be able to access it after exiting the page. For more information, see [API and Application Keys](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token).
```
I'd also move this to beneath step 3

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:47]:
```suggestion
1. Obtain an API key and Application key from [Datadog](https://app.datadoghq.com/organization-settings/api-keys) as described above.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:49]:
```suggestion
2. In DoControl, navigate to [Dashboard->Settings->Workflows->Secrets](https://app.docontrol.io/settings/workflows?tab=Secrets), and add your Datadog API key as a new secret.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:53]:
```suggestion
3. Create a new Workflow from a pre-established [**playbook**](https://app.docontrol.io/workflowV2/playbooks?filter=by_use_case&use_case=all) or create one from [**scratch**](https://app.docontrol.io/workflowV2/workflow/new/workflow-editor).
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:56]:
```suggestion
4. Design and edit your business logic by dragging and dropping actions onto the canvas, configuring the steps and connecting them.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:58]:
```suggestion
5. From the Actions bar, under **Utilities**, you can drag and drop Datadog actions, such as **Send logs** or **Create Incident**, into your Workflow. 
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:62]:
```suggestion
6. Configure the actions to refer to the DD-API-KEY stored as a secret in step #2 above, and the DD-APPLICATION-KEY obtained in step #1 above.
```

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:66]:
```suggestion
7. Learn more about DoControl in our [documentation](https://docs.docontrol.io/docontrol-user-guide/the-docontrol-console/workflows-beta/designing-and-editing-workflows/defining-workflow-and-action-settings#action-categories).
```

File: docontrol/images/IMAGES_README.md
Comment (anawishnoff) [inline docontrol/images/IMAGES_README.md:1]: This file can be deleted

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:18]: We recommend adding a couple images to the tile - these can be screenshots of the DoControl UI showing the Datadog Actions in a workflow, for example.

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:42]: Since you are not directly submitting any data into Datadog or pulling any data out, this section can be removed, and the `metadata.csv` and `service_checks.json` files can be deleted.

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Thanks for opening this PR! I've left an initial round of comments and suggestions.

File: docontrol/manifest.json
Comment (grosenthal1) [reply docontrol/manifest.json:42]: Which section should be deleted? Assets or Integration or metrics and service_checks?

Comment (anawishnoff) [reply docontrol/manifest.json:42]: The `integration` section can be removed, leaving `assets` as `{ }`

**Comment by kayayarai** [comment]
opened DOCS-3564 to track editorial review from docs team

File: docontrol/manifest.json
Comment (grosenthal1) [reply docontrol/manifest.json:22]: @anawishnoff I have added several categories aligned with our core competency.

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:27]: I don't think this screenshot is correct, as the yellow box is highlighting "log out" rather than "organization settings"

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:26]: Could we fix the typo in this screenshot? In the datadog box in the workflow it says "Log approval t DDOG" rather than "Log approval to DDOG"

File: docontrol/README.md
Comment (anawishnoff) [inline docontrol/README.md:62]:
```suggestion
6. Configure the actions to refer to the DD-API-KEY stored as a secret in Step 2 above, and the DD-APPLICATION-KEY obtained in Step 1 above.
```

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:25]:
```suggestion
        "caption": "Use DoControl's security automated workflows to create sophisticated, flexible and granular business logic with Datadog's log and incident capabilities.",
```

We try to avoid using marketing-heavy language for integrations listed on the Integrations page.

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:40]:
```suggestion
        "caption": "DoControl integrates with the most popular SaaS apps in multi-segments, including apps that fall into the categories of Collaboration, CRM, Communication, Dev tools, HR, IdP, EDR and more.",
```

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:50]:
```suggestion
        "caption": "DoControl's cross-SaaS anomaly detection mechanism triggers contextual alerts to reduce incident response time and provide auto-remediate measures.",
```

File: docontrol/manifest.json
Comment (anawishnoff) [inline docontrol/manifest.json:55]:
```suggestion
        "caption": "DoControl provides visibility into and enables remediation for the inventories of all your SaaS apps, including 3rd party apps (OAuth apps) - a major threat vector.",
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Leaving a few more comments but this is getting close on my end. Thanks!

File: docontrol/manifest.json
Comment (grosenthal1) [reply docontrol/manifest.json:26]: Fixed

**Review by anawishnoff — APPROVED** [review]
LGTM!

**Review by yzhan289 — APPROVED** [review]
LGTM! Once we get the DoControl logo ready internally we can merge this PR in. This should be good in a few days.

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for your PR! I've suggested some edits to align the page with our contributing guidelines:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md

Some of the larger edits revolve around:
- Avoiding duplication/redundancy of instructions
- Restructuring link format

Thanks again, and please feel free to reach out with any questions.

File: docontrol/README.md
Comment (grosenthal1) [reply docontrol/README.md]: Added

File: docontrol/README.md
Comment (buraizu) [inline docontrol/README.md:5]:
```suggestion
This integration allows [DoControl][9] customers to forward their DoControl-related logs and events to Datadog through automated security workflows.
```

File: docontrol/README.md
Comment (buraizu) [inline docontrol/README.md:9]:
```suggestion
To set up this integration, you must have an active [DoControl account][10]. You must also have proper [admin permissions][11] in Datadog.
```

Comment (buraizu) [reply docontrol/README.md:9]: Apologies for missing the link format issue on these in my first pass. Using reference links is helpful for the translation process, as mentioned in our contributing guidelines.
I've additionally added a link to the Datadog permissions section here. It's worth pointing out that `admin` permissions have been deprecated and replaced with more granular permission sets, as noted in the linked document. As such, it may be worthwhile to define which specific permission sets are required, but I'll leave that up to you.

File: docontrol/README.md
Comment (buraizu) [inline docontrol/README.md:23]:
```suggestion
1. Use Datadog's [Add an API key][2] documentation to create an API key. Give the key a meaningful name such as `DoControl`.
2. Copy the `Key` and save it. This key is not accessible after exiting the page.
```

File: docontrol/README.md
Comment (buraizu) [inline docontrol/README.md:52]:
```suggestion
   ![DC_DD_conf](https://raw.githubusercontent.com/DataDog/integrations-extras/master/docontrol/images/DC_DD_conf.png)
```

File: docontrol/README.md
Comment (buraizu) [inline docontrol/README.md:54]:
```suggestion
Learn more about DoControl in the [DoControl documentation][7].
```

Comment (buraizu) [reply docontrol/README.md:54]: Removing this from the list of steps as it's not required for setup.

File: docontrol/README.md
Comment (buraizu) [inline docontrol/README.md:71]:
```suggestion
[8]: mailto:support@docontrol.io
[9]: https://www.docontrol.io/
[10]: https://www.docontrol.io/demo
[11]: https://docs.datadoghq.com/account_management/rbac/permissions/
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thank you for applying those changes. After taking another look, I've realized that I missed updating the first two links to comply with our contributing guidelines. Apologies for that; I also unified the formatting in a couple of areas and added a link to Datadog permissions documentation.

**Comment by grosenthal1** [comment]
@buraizu seems like some checks are failing but those are at your side as it seems

**Comment by buraizu** [comment]
@grosenthal1 thanks for confirming. The updates look good and everything seems fine from a documentation perspective, but I'm just following up internally about the failing check. Thank you for your patience.

**Comment by buraizu** [comment]
@grosenthal1 thanks again for your patience. Just letting you know that a PR is open to address the failing check:
https://github.com/DataDog/integrations-extras/pull/1331

Once the PR is merged, could you rebase to allow the validations to pass? I'll follow up here as soon as this is confirmed on our end. Please feel free to reach out with any questions in the meantime.

**Comment by yzhan289** [comment]
Hi @grosenthal1, the PR fixing validation for the other integrations has been merged. You can rebase and push now!

**Review by yzhan289 — APPROVED** [review]

**Comment by yzhan289** [comment]
@grosenthal1 Just merged, you should see the new tile on prod in a few days.

**Pull request merged**
